File: Back/Domain/Product.cs
using Domain.Enums;

namespace Domain;
public class Product
{
    [Key]
    public string Id { get; set; }
    public required string Name { get; set; } 
    public required string ImageUrl { get; set; }
    public decimal Price { get; set; }
    public bool IsAvailable { get; set; }
    public UnitType UnitType { get; set; }
    
}

File: Front/TheShop/src/app/models/product.ts
export class Product {
  id: string;
  name: string;
  imageUrl: string;
  price: number;
  unitType?: string;

  constructor(id: string, name: string, imageUrl: string, price: number, unitType?: string) {
    this.id = id;
    this.name = name;
    this.imageUrl = imageUrl;
    this.price = price;
    this.unitType = unitType;
  }
}
